### PR TITLE
[W5.7][T10-2]Haw Zhe Hao Elroy

### PR DESCRIPTION
**Pull request by elroyhaw**
Sort enhancement to existing AddressBook

### CODE REVIEW

**Comment by YafeiY** [comment]
good enhancement :-)

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -73,33 +74,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
```
Comment (yunpengn) [inline src/seedu/addressbook/parser/Parser.java:76]: Please do not change the indentation level of switch. The original style is correct. See our [coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html).

image

**Review by yunpengn — APPROVED** [review]
Good idea. Please close this PR after viewing my comments.

File: src/seedu/addressbook/parser/Parser.java (hunk 1 above)
Comment (elroyhaw) [reply src/seedu/addressbook/parser/Parser.java:76]: Alright noted!

**Pull request closed without merging**